charts/retool/templates/_helpers.tpl

```diff
@@ -129,7 +129,7 @@ Usage: (include "retool.jobRunner.enabled" .)
 */}}
 {{- define "retool.jobRunner.enabled" -}}
 {{- $output := "" -}}
-{{- if or (gt (int (toString (.Values.replicaCount))) 1) (eq .Values.jobRunner.enabled true) }}
+{{- if or (gt (int (toString (.Values.replicaCount))) 1) (default false .Values.jobRunner.enabled) }}
```

Comment:
this was fine as `.Values.jobRunner.enabled` as well, but added default for clarity.

Reply:
so the "" and "1" output are fine, just the default needs to be changed?